install.sh

```diff
@@ -32,6 +32,9 @@ Usage:
   --version X.X.X
       Install a specific version instead of the latest.
 
+  --edge [true | false (default) ]
```

Comment:
Could we leave off the value and set this to true when the flag is set?  This is how we have been handling booleans, for example see `--dry-run`.